### ISSUE TITLE
font size < 7pt won't work on Linux

### ISSUE THREAD

**Issue by kmuto** [Trimmarks/versatype-issues#52]
It may be environment specific, but
font-size under 7pt always be ceiled to 7pt, on Debian GNU/Linux 8.5 x86_64 + Vivliostyle Formatter 2016.7.
In contrast OS X works as expected.

test code:

```
<html>
<head>
<meta charset="UTF-8" />
<style>
p { font-family: "A-OTF Gothic MB101 Pro", serif; }
</style>
</head>
<body>
<p style="font-size: 1pt; border-left: solid 1pt red;">ABCあいう</p>
<p style="font-size: 2pt; border-left: solid 2pt red;">ABCあいう</p>
<p style="font-size: 3pt; border-left: solid 3pt red;">ABCあいう</p>
<p style="font-size: 4pt; border-left: solid 4pt red;">ABCあいう</p>
<p style="font-size: 5pt; border-left: solid 5pt red;">ABCあいう</p>
<p style="font-size: 6pt; border-left: solid 6pt red;">ABCあいう</p>
<p style="font-size: 7pt; border-left: solid 7pt red;">ABCあいう</p>
<p style="font-size: 8pt; border-left: solid 8pt red;">ABCあいう</p>
<p style="font-size: 9pt; border-left: solid 9pt red;">ABCあいう</p>
<p style="font-size: 10pt; border-left: solid 10pt red;">ABCあいう</p>
</body>
</html>
```

Replacing font-family with others won't solve.

**Comment by skoji** [comment]
Sorry for the late reply.
I can not reproduce the problem on Ubuntu 14.04.2 + Vivliostyle Formatter 2016.7. 
I'll look further into this problem.

**Comment by skoji** [comment]
Confirmed on Debian GNU/Linux 8.5 x86_64.

**Comment by kmuto** [comment]
How is it fixed?

**Comment by skoji** [comment]
@kmuto Sorry, this issue is not fixed yet.

**Comment by skoji** [comment]
Fixed in version 2016.10 .
